From 86045727b7de06195786356fb3f1cc816b451b2d Mon Sep 17 00:00:00 2001
From: Han Xu <b45815@freescale.com>
Date: Tue, 9 Jun 2015 10:27:00 -0500
Subject: [PATCH 0780/1221] MLK-11074: ARM:dts: fix mipi pin conflict with
 ecspi1

mipi pins conflict with ecspi1 on i.MX7D 12x12 lpddr3 ARM2 board, manage
the mipi pins in a individual group to solve the issue.

Signed-off-by: Han Xu <b45815@freescale.com>
[zou:Original patch taken from
git.freescale.com/imx/fsl-arm-yocto-bsp.git-b imx-3.14.52-1.1.0_ga]
Signed-off-by: zou cao <cao.zou@windriver.com>
---
 arch/arm/boot/dts/imx7d-12x12-lpddr3-arm2.dts |    6 +++++-
 1 files changed, 5 insertions(+), 1 deletions(-)

diff --git a/arch/arm/boot/dts/imx7d-12x12-lpddr3-arm2.dts b/arch/arm/boot/dts/imx7d-12x12-lpddr3-arm2.dts
index 279c631..c50fbd1 100644
--- a/arch/arm/boot/dts/imx7d-12x12-lpddr3-arm2.dts
+++ b/arch/arm/boot/dts/imx7d-12x12-lpddr3-arm2.dts
@@ -252,7 +252,7 @@
 
 &iomuxc {
 	pinctrl-names = "default";
-	pinctrl-0 = <&pinctrl_hog_1 &pinctrl_hog &pinctrl_hog_sd2_vselect>;
+	pinctrl-0 = <&pinctrl_hog_1 &pinctrl_hog &pinctrl_hog_sd2_vselect &pinctrl_hog_mipi>;
 
 	hog {
 		pinctrl_hog_1: hoggrp-1 {
@@ -270,7 +270,11 @@
 				MX7D_PAD_SD2_CD_B__GPIO5_IO9      0x59
 				MX7D_PAD_SD2_WP__GPIO5_IO10       0x59
 				MX7D_PAD_SD2_RESET_B__GPIO5_IO11  0x59
+			>;
+		};
 
+		pinctrl_hog_mipi: hoggrp_mipi {
+			fsl,pins = <
 				MX7D_PAD_ECSPI1_SCLK__GPIO4_IO16  0x59
 				MX7D_PAD_ECSPI1_MOSI__GPIO4_IO17  0x59
 			>;
-- 
1.7.5.4

